### ISSUE TITLE
opinion on tweaked color theme

### ISSUE THREAD

**Issue by wibeasley** [National-COVID-Cohort-Collaborative/guide-to-n3c-v1#211]
Inspired by the title block stuff (#208), I peeked at the basic options for [title blocks](https://quarto.org/docs/authoring/title-blocks.html#title-banners).  (In my dev version), I added some color to the banner.  And also changed the hyperlinks from blue to N3C purple.

Any thoughts?  If someone wanted to tweak the css more, it's fine with me.

Proposed change:
![image](https://github.com/National-COVID-Cohort-Collaborative/guide-to-n3c-v1/assets/1372890/5542f5e6-ec1b-4324-bfd3-19b42b559d25)

Current version:
![image](https://github.com/National-COVID-Cohort-Collaborative/guide-to-n3c-v1/assets/1372890/f5b52f56-972e-4ce1-bb75-0932f9b4d879)

**Comment by chrisroederucdenver** [comment]
It breaks the header out nicely.  I like it.

**Comment by wibeasley** [comment]
I added a black diagonal gradient to it.  I like it a little better.  I tried to incorporate the logo, but I couldn't get it to look right when the page was resizing.

@oneilsh & @chrisroederucdenver, I'm going to pull it into the main branch, but feel free to undo it if it wears on you.  

![image](https://github.com/National-COVID-Cohort-Collaborative/guide-to-n3c-v1/assets/1372890/78538224-0bff-4bde-b4c3-0a7f0196117f)
